src/main/java/com/openshift/jenkins/plugins/pipeline/MessageConstants.java

```diff
@@ -0,0 +1,60 @@
+package com.openshift.jenkins.plugins.pipeline;
+
+public class MessageConstants {
+
+public static final String CANNOT_GET_CLIENT = "\n\nExiting \"%s\" unsuccessfully; a client connection to \"%s\" could not be obtained.";
+public static final String BAD_BUILD_STATE = "\n\nExiting \"%s\" unsuccessfully; build \"%s\" has completed with status:  [%s].";
+public static final String GOOD_BUILD_WITH_DEPLOY = "\n\nExiting \"%s\" successfully; build \"%s\" has completed with status:  [Complete].  All deployments with ImageChange triggers based on this build's output triggered off of the new image.";
+public static final String GOOD_BUILD_BAD_DEPLOY = "\n\nExiting \"%s\" unsuccessfully; build \"%s\" has completed with status:  [Complete]. However, not all deployments with ImageChange triggers based on this build's output triggered off of the new image.";
+public static final String GOOD_BUILD_IGNORE_DEPLOY = "\n\nExiting \"%s\" successfully; build \"%s\" has completed with status:  [Complete].";
+public static final String START_BUILD_CANCEL = "\n\nStarting the \"%s\" action for build config \"%s\" from the project \"%s\".";
+public static final String CANCEL_SINGLE_BUILD = "  Cancelled build \"%s\".";
+public static final String EXIT_BUILD_CANCEL = "\n\nExiting \"%s\" successfully with %d builds cancelled.";
+public static final String START_BUILD = "\n\nStarting the \"%s\" step with build config \"%s\" from the project \"%s\".";
+public static final String BUILD_CANNOT_START = "\n\nExiting \"%s\" unsuccessfully; a build could not be started";
+public static final String BUILD_STARTED = "  Started build \"%s\" and waiting for build completion %s...";
+public static final String BUILD_WAIT_DEPLOY = "followed by a new deployment";
+public static final String BUILD_NO_POD = "\n\nExiting \"%s\" unsuccessfully; the build pod for build \"%s\" was not found in time.";
+public static final String BUILD_NO_CFG = "\n\nExiting \"%s\" unsuccessfully; the build config \"%s\" could not be read.";
+public static final String BUILD_VERIFYING = "  Verifying build \"%s\" and waiting for build completion %s...";
```

Comment:
Verifying the build completes successfully, and that optionally that the appropriate deployments fired (the last %s will either be an empty string or a string re: the deployments).

How's this alternative for the message itself:  "  Waiting on build \"%s\" to complete %s..."

For the constant name  ... "WAIT_ON_BUILD" perhaps ?